tsl/src/chunk_copy.c

```diff
@@ -399,7 +416,7 @@ chunk_copy_stage_create_publication(ChunkCopy *cc)
 				   quote_qualified_identifier(NameStr(cc->chunk->fd.schema_name),
 											  NameStr(cc->chunk->fd.table_name)));
 
-	/* Create the publication in autocommit mode */
+	/* Create the publication */
 	ts_dist_cmd_run_on_data_nodes(cmd, list_make1(NameStr(cc->fd.source_node_name)), true);
```

Comment:
I am wondering if we should execute those commands in the auto-commit mode since we check if the object exist anyway during the cleanup

Reply:
We are not trying to speed things up here. We might add a AN controlled 1PC transaction if only one datanode is involved in the future. So, I would like to keep things controlled via AN as much as possible.